projects/1.20/assets/medieval-origins-revival/medievalorigins/lang/zh_cn.json

```
@@ -0,0 +1,425 @@
+{
+  "origin.medievalorigins.arachnae.name": "阿拉克涅",
+  "origin.medievalorigins.arachnae.description": "阿拉克涅（名字来自罗马神话被变成蜘蛛的那个下面就都这样叫了）拥有人类的力量和蜘蛛的灵巧与狡诈。是真正可怕的猎手。",

```

```suggestion
  "origin.medievalorigins.arachnae.description": "阿拉克涅拥有人类的力量和蜘蛛的灵巧与狡诈。是真正可怕的猎手。",
```
不可随意加注释，特别是这种仅作审核时标记使用的

顺便阿拉克涅应该是希腊神话的内容，阿拉克涅这个单词本身就是希腊语的“蜘蛛”